wsi/drm/drmplane.cpp

```diff
@@ -418,6 +451,21 @@ void DrmPlane::SetInUse(bool in_use) {
   in_use_ = in_use;
 }
 
+bool DrmPlane::IsSupportedModifier(uint64_t modifier, uint32_t format) {
+  uint32_t count = formats_modifiers_.size();
+  for (int32_t i = 0; i < count; i++) {
```

Comment:
we should take the format as input and return supported format here

Reply:
Default should be DRM_FORMAT_MOD_NONE

Reply:
Given a format there should be several modifiers supported. you mean return all supported modifiers?

Reply:
hmm right. Leave it as it is for now